webots_ros2_tutorials/webots_ros2_tutorials/master.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 1996-2020 Soft_illusion.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import Float64
+from geometry_msgs.msg import Twist
+
+
+class LineFollower(Node):
+    def __init__(self):
+        super().__init__('linefollower_cmdvel')
+        # Subscribe IR sensors
```

Comment:
The meaning of IR, GS is not clear, it would be better to avoid abbreviations.

Reply:
changed the names of variables